contracts/protocol/libraries/position/PositionHouseFunction.sol

```diff
@@ -794,7 +794,7 @@ library PositionHouseFunction {
         );
 
         positionResp.realizedPnl =
-            (unrealizedPnl * int256(positionResp.exchangedPositionSize)) /
+            (unrealizedPnl * positionResp.exchangedPositionSize.absInt()) /
```

Comment:
this should be the absolute value of `exchangedPositionSize` cause `exchangedPositionSize` can be negative, we just calculate based on the percentage of quantity